components/automate-ui/src/app/pages/create-notification-modal/create-notification-modal.component.html

```diff
@@ -0,0 +1,126 @@
+                panelClass="chef-dropdown"
+                formControlName="targetType"
+                placeholder="Webhook Type"
+                (selectionChange)="changeSelectionForWebhookType($event)">
+                <mat-option *ngFor="let key of targetKeysValue" [value]="key">
+                  <img src="assets/img/logo-{{key}}.png" alt="{{key}} logo"/>
+                  {{ notificationRule.TargetTypeLabels[key] }}
+                </mat-option>
+              </mat-select>
+            </label>
+          </chef-form-field>
+        </div>
+        <div class="margin">
+          <chef-form-field>
+            <label>
+              <span class="label">Failure type  <span aria-hidden="true">*</span></span>
+              <mat-select
+                panelClass="chef-dropdown"
+                formControlName="ruleType"
+                placeholder="Failure Type"
+                (selectionChange)="setFailureType($event)">
+                <mat-option *ngFor="let key of alertTypeKeysValue" [value]="key">
+                  {{ notificationRule.AlertTypeLabels[key] }}
+                </mat-option>
+              </mat-select>
+              <div *ngIf="displayCriticalControlsCheckbox()">
+                <p class="small">
+                  <chef-checkbox
+                    (change)="updateCriticalControlsOnly($event.detail)"
+                    [checked]="notificationRule.criticalControlsOnly">
+                  </chef-checkbox>
+                  Only send notifications for critical control failures
+                </p>
+              </div>
+            </label>
+          </chef-form-field>
+        </div>
+        <div class="margin">
+          <chef-form-field>
+            <label>
+              <span class="label">Webhook URL <span aria-hidden="true">*</span></span>
+              <input chefInput formControlName="targetUrl" type="text" id="targetUrl-input" (keyup)="handleInput($event)" autocomplete="off"/>
+            </label>
+            <chef-error
+            *ngIf="(createForm.get('targetUrl').hasError('required') || createForm.get('targetUrl').hasError('pattern')) && createForm.get('targetUrl').dirty">
+            Webhook URL is required.
+            </chef-error>
+          </chef-form-field>
+        </div>
+        <div *ngIf="createForm.value.targetType === 'ServiceNowAlert'">
+          <div class="margin">
+            <chef-form-field>
+              <label>
+                <span class="label">ServiceNow Username <span aria-hidden="true">*</span></span>
+                <input chefInput formControlName="username" type="text" id="username-input" (keyup)="handleInput($event)" autocomplete="off"/>
+              </label>
+              <chef-error *ngIf="createForm.get('username').hasError('required') && createForm.get('username').dirty">
+                Username is required.
+              </chef-error>
+              <span class="i-desc">Encrypted, and only used to access the notification.</span>
```

Comment:
I assume this message should be the same as that on L97?
```suggestion
              <span class="i-desc">Encrypted, and only used to access the notification feed.</span>
```